plugin/chatgpt.vim

```diff
@@ -218,23 +232,10 @@ function! SendHighlightedCodeToChatGPT(ask, context)
     let yanked_text = '```' . syntax . "\n" . @@ . "\n" . '```'
   endif
 
-  let prompt_templates = {
-  \ 'rewrite': 'Can you rewrite it more idiomatically?',
-  \ 'review': 'Can you provide a code review for?',
-  \ 'document': 'Return documentation following language pattern conventions.',
-  \ 'explain': 'Can you explain it?',
-  \ 'test': 'Can you write a test for it?',
-  \ 'fix': 'It has an error I need you to fix.'
-  \}
-
-  if exists('g:chat_gpt_custom_prompts')
-    call extend(prompt_templates, g:chat_gpt_custom_prompts)
-  endif
-
   let prompt = a:context . ' ' . "\n" . yanked_text
 
-  if has_key(prompt_templates, a:ask)
-    let template  = "Given the following code snippet ". prompt_templates[a:ask]
+  if has_key(g:prompt_templates, a:ask)
+    let template  = "Given the following code snippet ". g:prompt_templates[a:ask]
```

Comment:
I would remove the "Given the following code snippet " from the prompt as this might produce nonsensical prompts with user prompts

Reply:
Make sense i've updated the wrapper to only include that when highlighted text is sent.